### PR TITLE
API: read (env) variable `TLAPM_CACHE_DIR`, add command-line parameter `--cache-dir`

### PR DESCRIPTION
**Pull request by johnyf**
Addresses #16.

### CODE REVIEW

File: src/tlapm_args.ml [hunk 1]
```diff
@@ -212,6 +212,18 @@ let init () =
...
+             temporary files under <directory>. \
+             Alternatively, this directory \
+             can be defined via the variable \
+             `TLAPM_CACHE_DIR` of the runtime \
+             environment. For example, by \
+             writing in `$HOME/.bashrc` the line \
+             `export TLAPM_CACHE_DIR=\
+             'cache_dir_name'. The command-line \
+             parameter takes precedence over \
+             the environment variable.";
```
Comment (damiendoligez) [inline src/tlapm_args.ml:226]:
```suggestion
             environment. The command-line \
             parameter takes precedence over \
             the environment variable. If neither \
             is specified, the default value is \".tlacache\".";
```

**Review by damiendoligez — APPROVED** [review]
Looks OK modulo a small suggestion for a change to the doc string.

**Comment by johnyf** [comment]
Thank you for reviewing and approving the changes, I will apply the suggested change and merge the pull request.

File: src/tlapm_args.ml (hunk 1 above)
Comment (johnyf) [reply src/tlapm_args.ml:226]: Thank you for suggesting this change, I will rebase to update the docstring.

**Pull request merged**